VideoLocker/src/main/java/org/edx/mobile/view/custom/URLInterceptorWebViewClient.java

```diff
@@ -166,6 +167,7 @@ public WebResourceResponse shouldInterceptRequest(WebView view, String url) {
 
             // suppress external links on ZeroRated network
             if (isExternalLink(url)
+                    && !ConfigUtil.isWhiteListedURL(url)
```

Comment:
@aleffert  this is used only for background http request triggered by html code.   the logic is only used in this class.